### DISCUSSION TITLE
✅ 🚀 Feature Request: Provide more design controls over Dropdown Indicators in the Mega Menu widget

### DISCUSSION THREAD

**Discussion by Joliamendi** [elementor/elementor#21907] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi team

The mega menu is a super powerful feature you added and it's really cool. The mega menu is a super powerful feature you've added and it's really cool. But there is nothing to customize the **Dropdown Indicator** and it is forgotten. Currently, there is only custom icon feature for them. only this!

### Describe the solution you'd like

Please add style customization options to Dropdown Indicator. **Required customization options :**

1. spacing (Control the distance between the Dropdown Indicator and the text)
2. size (Currently, its size is tied to the size of the text)
3. Color control (in normal, hover and active status)
4. transition (Ability to control the speed of icon change on hover)
5. icon position (Ability to move the Dropdown Indicator icon to the right or left side of the text)   This is very useful in some situations.

thnks

**Comment by nicholaszein** [comment]
Hello @Joliamendi,

thanks for submitting your feature request! 🙏 

Please understand that while we do welcome new feature requests, we can't promise they will be considered or added to our roadmap for the time being. 

🔖 We will, of course, keep this suggestion in mind and consider adding it if there is significant demand in the future.
Best regards

**Comment by nicholaszein** [comment]
Hey @Joliamendi!

I'm not sure if you know this, but we added styling options for the Dropdown Indicator in the mega menu in version 3.19.0.
You can find those options in a new section in the Style tab of the (nested) Menu called "Dropdown indicator".

Thank you for your contribution!

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
